src/cgutils.cpp

```diff
@@ -998,37 +998,37 @@ static void emit_memcpy_llvm(jl_codectx_t &ctx, Value *dst, jl_aliasinfo_t const
     // above problem won't be as serious.
 
     auto merged_ai = dst_ai.merge(src_ai);
-    ctx.builder.CreateMemCpy(dst, MaybeAlign(align), src, MaybeAlign(0), sz, is_volatile,
+    ctx.builder.CreateMemCpy(dst, MaybeAlign(align_dst), src, MaybeAlign(align_src), sz, is_volatile,
```

Comment:
```suggestion
    ctx.builder.CreateMemCpy(dst, Align(align_dst), src, Align(align_src), sz, is_volatile,
```
Seems like we should be able to avoid passing `0` (esp since we asserted it above for `align_dst`)

Reply:
So there are a couple of places where i'm not sure yet, so I haven't changed this yet.